multimap/src/main/java/org/infinispan/multimap/api/embedded/MultimapCacheManager.java

```diff
@@ -25,5 +25,17 @@
     * @return null if no configuration exists as per rules set above, otherwise returns a multimap cache instance
     * identified by cacheName
     */
-   MultimapCache<K, V> get(String name);
+   default MultimapCache<K, V> get(String name) {
```

Comment:
Same thing regarding documentation that the returned multi map cache does not support duplicates.